system/classes/Kohana/File.php

```diff
@@ -139,7 +139,11 @@ public static function exts_by_mime($type)
 	 */
 	public static function ext_by_mime($type)
 	{
-		return current(File::exts_by_mime($type));
+		$exts = File::exts_by_mime($type);
+		if (!$exts) {
```

Comment:
Please put curly brackets to separate line according to https://docs.koseven.ga/guide/kohana/conventions

Reply:
Done, thanks for the review